src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,28 @@
+.stars {
+  @include zero-margins-paddings;
+  display: flex;
+  list-style: none;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    display: flex;
+    margin-left: 4px;
+    background-image: url(/images/star-active.svg);
+    background-position: center;
+    background-repeat: no-repeat;
```

Comment:
1. Not sure that you need `display: flex` here
2. You can combine all background properties like this `background: url() center no-repeat`